src/transaction/log_replication_atomic.hpp

```diff
@@ -98,6 +101,11 @@ namespace cublog
 
       log_lsa m_lowest_unapplied_lsa;
       mutable std::mutex m_lowest_unapplied_lsa_mutex;
+
+      /* Store the locked objects for DDL replication.
+       * Since multiple DDL operations can be executed within single transaction,
+       * more than one objects can be mapped to one transaction */
+      std::multimap <TRANID, OID> m_locked_objects;
```

Comment:
Let's make it clear it's classes.
```suggestion
      std::multimap <TRANID, OID> m_locked_classes; 
```

Reply:
OK, thanks.